rotkehlchen/api/rest.py

```diff
@@ -4416,3 +4424,29 @@ def search_for_names_everywhere(self, addresses: List[ChecksumEthAddress]) -> Re
             addresses=addresses,
         )
         return api_response(_wrap_in_ok_result(mappings))
+
+    def get_config_arguments(self) -> Response:
```

Comment:
As far as I know `args` are not modified at any point, so why not using `rotkehlchen.args` directly instead of creating a new dictionary that holds the same data?

Reply:
yeah I agree with Alexey here. Why not just use the `rotkehlchen.args` object? Why do you need the extra dict?